editor/components/writing-flow/index.js

```diff
@@ -98,6 +98,33 @@ class WritingFlow extends Component {
 		} );
 	}
 
+	isInLastNonEmptyBlock( target ) {
+		const tabbables = this.getVisibleTabbables();
+
+		// Find last tabbable, compare with target
+		const lastTabbable = last( tabbables );
+		if ( ! lastTabbable || ! lastTabbable.contains( target ) ) {
+			return false;
+		}
+
+		// Find block-level ancestor of said last tabbable
+		const blockEl = lastTabbable.closest( '.editor-block-list__block-edit' );
```

Comment:
Should we use a const for the classname as it's being used in several places.

Reply:
@youknowriad, what do you think of 31f9e0a35eb7a46b7749e12baf35779dfefa72b9? As an alternative to spinning up a `constants` file, etc.

Reply:
I love it